openupgradelib/openupgrade_160.py

```diff
@@ -385,21 +385,23 @@ def _convert_field_bootstrap_4to5_sql(cr, table, field, ids=None):
     :param list ids:
         List of IDs, to restrict operation to them.
     """
-    sql = "SELECT id, %s FROM %s " % (field, table)
-    params = ()
+    sql = "SELECT id, %s FROM %s "
+    params = (AsIs(field), AsIs(table))
```

Comment:
In fact, this should be `sql.Identifier`, and built with `sql.SQL`

Reply:
Done :+1: